acceptance/bundle/deployment/bind/dashboard/script

```diff
@@ -0,0 +1,43 @@
+title "Bind dashboard test: "
+
+title "Substitute variables in the template: "
+BUNDLE_NAME_SUFFIX=$(uuid)
+export BUNDLE_NAME_SUFFIX
+
+DASHBOARD_DISPLAY_NAME="test dashboard $(uuid)"
+if [ -z "$CLOUD_ENV" ]; then
+    DASHBOARD_DISPLAY_NAME="test dashboard 6260d50f-e8ff-4905-8f28-812345678903"   # use hard-coded uuid when running locally
+    export TEST_DEFAULT_WAREHOUSE_ID="warehouse-1234"
```

Comment:
Is there any downside for hardcoding these in acceptance_test.go if CLOUD_ENV is "" and they are not set already?

Reply:
the only thing i can think of - this style keeps this local to the test and does not encourage variables that are too use-case specific to be used in unrelated tests